website/docs/developer/overview.md

```diff
@@ -0,0 +1,16 @@
+---
+id: overview
+title: Overview
+sidebar_label: Overview
+---
+
+OpenAttestation comes with a set of tools that help you to build, verify or share documents depending on your need.
+
+In this section you will discover all the tools that OpenAttestation offers and how to use them:
+
+- [Wrapper SDK (javascript)](/docs/developer/open-attestation) is the core of OA allowing you to create transferable record or verifiable document.
```

Comment:
should we rename the links as what the titles is?
```suggestion
- [open-attestation (javascript)](/docs/developer/open-attestation) is the core of OA allowing you to create transferable record or verifiable document.
```
same for the rest

Reply:
ok done.